lambda-layer/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+pushd src || exit
+rm -rf build
+./build-lambda-layer.sh
+popd || exit
+
+pushd sample-apps || exit
+rm -rf build
+./package-lambda-function.sh
+popd || exit
+
```

Comment:
Q: why does the `sample-apps` need to be built while building the layer artifact?

Reply:
oh. this is not for only building lambda layer? it is to build everything for testing lambda?

Reply:
Correct, it build and deploy layer with a sampled app.